apps/site/lib/site/shuttle_diversion.ex

```diff
@@ -10,12 +10,13 @@ defmodule Site.ShuttleDiversion do
   defmodule Shape do
     @moduledoc "Derivative of an API `shape` tailored for drawing a shuttle diversion map."
 
-    @enforce_keys [:id, :direction_id, :polyline, :is_shuttle_route]
+    @enforce_keys [:id, :direction_id, :color, :polyline, :is_shuttle_route]
     defstruct @enforce_keys
 
     @type t :: %__MODULE__{
             id: String.t(),
             direction_id: 0 | 1,
+            color: String.t(),
```

Comment:
Sorry for the crossed wires, but I merged a PR last week that was supposed to serve as the source for the line color, just in a different place: https://github.com/mbta/dotcom/pull/289

Thoughts on this approach vs. that one?

Reply:
I'm not clear how #289 offers an alternate way of getting the color onto the `Shape`? Or was the intended approach to not put the color onto the shape, but use some other way?

Reply:
It adds a `color` attribute onto Routes, and we have the current Route available on the shuttles page. So: for each shape, if this is not a shuttle shape, use the color defined on the current route, else use a frontend-defined "shuttle color".

Reply:
I can see how this way might be easier since it places the color closer to the shape itself, which is where we eventually want it for the `Map`, and the `color` defined in the API for the shuttle shapes happens to be the color we want to use on the map. In that case maybe we should revert #289 since it now serves no purpose.

Reply:
😅 I also ended up doing that because I wasn't finding any color attribute on the route. But I try it now, and it's there... whoops. I'd still need to attach the color to the shape, but I think it might be easier for me to do that in the same manner I attach the headsign onto the stop (for the tooltip), so I think I'll do that and revert this extra stuff.

Reply:
This has been corrected, thank you!